src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js

```diff
@@ -1,14 +1,30 @@
 import { Header } from 'semantic-ui-react';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import _isEmpty from 'lodash/isEmpty';
 
 export class DocumentTitle extends Component {
+  subtitle() {
+    const {
+      metadata: { alternative_titles },
+    } = this.props;
+    if (!_isEmpty(alternative_titles)) {
+      const subtitle = alternative_titles.find(e => e.type === 'SUBTITLE');
+      if (!_isEmpty(subtitle)) {
+        return <Header.Subheader>{subtitle.value}</Header.Subheader>;
+      }
+    }
+  }
+
   render() {
     const { metadata } = this.props;
     return (
       <>
         {metadata.document_type}
-        <Header as="h2">{metadata.title}</Header>
+        <Header as="h2" className="item-title">
```

Comment:
not blocking: could you change the class name ? it is in fact document title not item title so the name is confusing. thanks!